spec/rubocop/cop/style/negated_if_spec.rb

```diff
@@ -95,6 +95,21 @@
 
       expect(cop.offenses).to be_empty
     end
+
+    it 'autocorrects by replacing if not with unless' do
```

Comment:
Here probably there should be tests for both `prefix` and `post` auto-correction.